sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/EdgeNGramTokenFilter.java

```diff
@@ -36,6 +32,13 @@
     @JsonProperty(value = "side")
     private EdgeNGramTokenFilterSide side;
 
+    /**
+     * Constructor for {@link EdgeNGramTokenFilter}.
+     */
+    public EdgeNGramTokenFilter() {
+        odataType = "#Microsoft.Azure.Search.EdgeNGramTokenFilterV2";
```

Comment:
Could we make this String value a constant, I believe making the implementation constant public would work. Same comment for other places where we are doing this.